Source/UI/NewUI/CommonUI/View/Component/InputView.swift

```diff
@@ -7,6 +7,7 @@ class InputView: UIView {
     weak var delegate: TextFieldViewDelegate?
     private(set) var style: CellTextFieldStyle?
     private(set) lazy var textFieldContainerBottomAnchor = textFieldContainer.bottomAnchor.constraint(equalTo: bottomAnchor)
+    private(set) lazy var iconViewWidthAnchor = iconView.widthAnchor.constraint(equalToConstant: 32)
```

Comment:
Might be handy to set that 32 inside the Constants. Would even question if we're sure we want it to always be 32, as there seems to be no other logic to ever change it.

Reply:
When I add in customisation this will be moved to constants